src/components/NextQuestion/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+
+import { SurveyAnswerContext } from 'contexts/surveyAnswer';
+import nextIcon from 'images/next-button-black.svg';
+
+const NextQuestion = ({ questionId, nextQuestion }) => {
+  const answerContext = useContext(SurveyAnswerContext)
+  const answerObject = answerContext.state.questions.find(obj => obj.id === questionId);
+
+  // Checking if the answer is selected for this question
+  // if not does not go to next question
+  const answerSelected = () => {
+    if (answerObject && answerObject.answers.length !== 0) {
+      nextQuestion()
+    } else {
+      return false;
+    }
+  }
+
+  return(
+    <React.Fragment>
+      <div
+        className="next-question"
+        role="presentation"
+        onClick={ answerSelected }
+        data-test-id="next-question"
```

Comment:
How about use camelCase for the `data-test-id` to follow our convention as it is still an `id`.
![Screen Shot 2564-06-18 at 12 00 50](https://user-images.githubusercontent.com/29707647/122508828-d4aa4880-d02c-11eb-8ca8-ec2594ab76e6.png)

Reply:
As you use kebab case for most of them at the moment. If you agree with my comment above, you can create another task to fix this.